recipes/pysqa/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pysqa" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d8d07a0e8a63bf04237806d3cd1a4dc4e3613230109c486b289e23c87cfb155b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - defusedxml
+    - pandas
+    - yaml
+    - jinja2
+  run:
+    - python
+    - defusedxml
+    - pandas
+    - yaml
+    - jinja2
+
+test:
+  imports:
+    - pysqa
+
+about:
+  home: https://github.com/pysqa/pysqa
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Simple queue adapter for Python'
+
+  description: |
```

Comment:
If there isn't a longer description for the package this section can be removed (along with the blank line above).